packages/infer-schema/.npmrc

```diff
@@ -0,0 +1 @@
+save-exact=true
```

Comment:
I just copied what the `packages/graphql/.npmrc` had.

Reply:
If there was a confused emoji, I would apply it to the above comment... 🧐

Reply:
Confused by my comment? 
I just meant that I copied the configuration from the `packages/graphql` package and didn't think about what was in the dot files (or jest or ts configs either).
Are you saying that we should go with the default (i.e. not `save-exact=true`) in all packages?

Reply:
Sorry, not by your comment, just to the fact that I didn't realise that was there!

I have a feeling we should remove that, because our dependencies aren't saved as exact versions in our `package.json` files anyway.